cli/src/command/account.rs

```diff
@@ -992,3 +1002,32 @@ async fn print_address(account: &Account, address: &AccountAddress) -> Result<()
 
     Ok(())
 }
+
+async fn print_outputs(mut outputs: Vec<OutputData>, title: &str) -> Result<(), Error> {
+    outputs.sort_by(|a, b| {
+        a.metadata
+            .milestone_timestamp_booked()
+            .cmp(&b.metadata.milestone_timestamp_booked())
+    });
+
+    if outputs.is_empty() {
+        println_log_info!("No outputs found");
+    } else {
+        println_log_info!("{title}");
+        for (i, output_data) in outputs.into_iter().rev().enumerate() {
+            let booked_time = to_utc_date_time(output_data.metadata.milestone_timestamp_booked() as u128 * 1000)?;
+            let formatted_time = booked_time.format("%Y-%m-%d %H:%M:%S UTC").to_string();
+
+            println_log_info!(
+                "{:<5}{}\t{}\t{}\t{}",
+                i,
+                &output_data.output_id,
+                output_data.output.kind_str(),
+                if output_data.is_spent { "spent" } else { "unspent" },
```

Comment:
```suggestion
                if output_data.is_spent { "Spent" } else { "Unspent" },
```
Looks a bit better IMO, feel free to close. Also I think I would put it last